code/game/turfs/open/space/space.dm

```diff
@@ -50,6 +50,9 @@
 			smoothing_flags |= SMOOTH_OBJ
 		SET_BITFLAG_LIST(canSmoothWith)
 
+	var/area/our_area = loc
+	if(our_area.static_lighting && always_lit) //Only provide your own lighting if the area doesn't for you
+		add_overlay(GLOB.fullbright_overlay)
 
```

Comment:
this logic basically says
"If this area uses lighting underlays and is always lit only then apply the underlay"
are you sure you arent looking for base_lighting

Reply:
Yes. I'm trying to replicate this behavior
https://github.com/TiviPlus/tgstation/blob/d94f892f65154428cff61494a7aa70347baddc44/code/game/turfs/turf.dm#L111-L113

So: If the area isn't producing any light of its own and we have light, apply the overlay.

Maybe I should be using `area_has_base_lighting`? It's odd since you kept behavior reasonably static elsewhere, but changed it significantly here. Was it to support colored area lights or something?

Reply:
yea use area_has_base_lighting